airbyte-integrations/connectors/source-open-exchange-rates/source_open_exchange_rates/schemas/open_exchange_rates_stream.json

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
This file is not correctly named: it should match the class name, in your case that would give: `open_exchange_rates.json`

(see [the doc about schema files](https://docs.airbyte.com/connector-development/tutorials/cdk-tutorial-python-http/declare-schema ))

Reply:
Done